server/src/pages/homepage/view.js

```diff
@@ -101,10 +103,8 @@ class Body extends React.Component {
                   <a href="#" className="screenshots-logo"></a>
                 </div>
                 <div className="nav-links">
-                  <Localized id="homePageGetStarted">
-                    <a href="#how-screenshots-works" onClick={ this.onClickGetStarted.bind(this) }>Get Started</a>
-                  </Localized>
```

Comment:
If we are removing this anchor, then the handler, content, etc. should be removed as well.

Reply:
Oh, the l10n string for `homePageGetStarted` too.

Reply:
Done

Reply:
@johngruen In today's meeting we agreed to start merging FxA patches into master, so I would like your take on this change.  It removes the Get Started nav and content from the home page.

Reply:
@punamdahiya The content is still here.

Reply:
@chenba by content you mean below text on homepage. I believe that should be removed as part of homepage re-design @johngruen wdyt?
![screenshot_2018-08-30 firefox screenshots](https://user-images.githubusercontent.com/597734/44876566-52bbbb80-ac56-11e8-8fb8-2e24c8e12a21.png)

Reply:
@punamdahiya hmm yeah, that'd be better.  Let's leave the page content alone then.